lib/src/evaluateFlags.test.ts

```diff
@@ -39,5 +41,12 @@ describe("evaluateFlags", () => {
     expect(result).toEqual({
       toggles: [],
     });
+
+    expect(console.error).toHaveBeenCalledWith(
```

Comment:
I don't see explanation for this change

Reply:
Decreasing noise in test output - without this it spills an error when running tests.